ts_src/transaction_builder.ts

```diff
@@ -502,6 +505,23 @@ function expandInput(
         maxSignatures: m,
       };
     }
+
```

Comment:
Duplicate processing in case `SCRIPT_TYPES.P2MS`

Reply:
Removed at 4ccef81d1a6a6b2f79129e9b6ec57fa7855c200f